commit 1939f165e5f5fdb1ed2ed8df3577f3fc5a828dcb (refs/changes/32/3332/2)
Author: Julien Gilli <julien.gilli@joyent.com>
Date:   2018-02-08T14:15:26-08:00 (1 year, 8 months ago)
    
    PUBAPI-1470 volume objects should expose their creation timestamp in a property named "created" instead of "create_timestamp"

diff --git a/CHANGES.md b/CHANGES.md
index cecd8a0..f6332c5 100644
--- a/CHANGES.md
+++ b/CHANGES.md
@@ -6,7 +6,10 @@ Known issues:
 
 ## not yet released
 
-(nothing yet)
+## 5.6.1
+
+- [PUBAPI-1470] volume objects should expose their creation timestamp in a
+  property named "created" instead of "create_timestamp".
 
 ## 5.6.0
 
diff --git a/lib/do_volume/do_list.js b/lib/do_volume/do_list.js
index 147650f..85bab4e 100644
--- a/lib/do_volume/do_list.js
+++ b/lib/do_volume/do_list.js
@@ -96,14 +96,13 @@ function do_list(subcmd, opts, args, callback) {
                         var created;
                         var volume = volumes[i];
 
-                        created = new Date(volume.create_timestamp);
+                        created = new Date(volume.created);
 
                         if (volume.filesystem_path !== undefined) {
                             volume.resource = volume.filesystem_path;
                         }
 
                         volume.shortid = volume.id.split('-', 1)[0];
-                        volume.created = volume.create_timestamp;
                         volume.age = common.longAgo(created, now);
                     }
 
diff --git a/package.json b/package.json
index 1096f50..0d00f3f 100644
--- a/package.json
+++ b/package.json
@@ -1,7 +1,7 @@
 {
   "name": "triton",
   "description": "Joyent Triton CLI and client (https://www.joyent.com/triton)",
-  "version": "5.6.0",
+  "version": "5.6.1",
   "author": "Joyent (joyent.com)",
   "homepage": "https://github.com/joyent/node-triton",
   "dependencies": {
